OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -210,6 +210,7 @@
     <Compile Include="Traits\ExternalCapturable.cs" />
     <Compile Include="Traits\ExternalCapturableBar.cs" />
     <Compile Include="Traits\ExternalCaptures.cs" />
+    <Compile Include="Traits\IgnoresCloak.cs" />
```

Comment:
I'm pretty sure you don't need this anymore either because of the way the target selection works now.